File: model/Paciente.php
<?php
    require_once 'Pessoa.php';
    class Paciente extends Pessoa{
        private $estado;
        private $cidade;
        private $medicacao;
        private $doenca;
        private $tipo_sanguineo;

        public function setEstado($estado){
            $this->estado = $estado;
        }

        public function getEstado(){
            return $this->estado;
        }

        public function setCidade($cidade){
            $this->cidade = $cidade;
        }

        public function getCidade(){
            return $this->cidade;
        }

        public function setMedicacao($medicacao){
            $this->medicacao = $medicacao;
        }

        public function getMedicacao(){
            return $this->medicacao;
        }

        public function setDoenca($doenca){
            $this->doenca = $doenca;
        }

        public function getDoenca(){
            return $this->doenca;
        }

        public function setTipoSanguineo($tipo_sanguineo){
            $this->tipo_sanguineo = $tipo_sanguineo;
        }

        public function getTipoSanguineo(){
            return $this->tipo_sanguineo;
        }
    }

?>

File: java/src/model/Paciente.java
package model;

public class Paciente extends Pessoa{
    private String nome_social;
    private String medicacao;
    private String doenca;
    private String tipo_sanguineo;

    public Paciente(){};

    public Paciente(int id, String nome, String email, String senha){
        super(id, nome, email, senha);
    }

    public String getNomeSocial() {
        return this.nome_social;
    }

    public void setNomeSocial(String nome_social) {
        this.nome_social = nome_social;
    }

    public String getMedicacao() {
        return this.medicacao;
    }

    public void setMedicacao(String medicacao) {
        this.medicacao = medicacao;
    }

    public String getDoenca() {
        return this.doenca;
    }

    public void setDoenca(String doenca) {
        this.doenca = doenca;
    }

    public String getTipoSanguineo() {
        return this.tipo_sanguineo;
    }

    public void setTipoSanguineo(String tipo_sanguineo) {
        this.tipo_sanguineo = tipo_sanguineo;
    }

    @Override
    public String toString(){
        return super.toString()+". Nome social: "+this.nome_social+". Medicações: "+this.medicacao+". Doenças: "+this.doenca+". Tipo sanguíneo: "+this.tipo_sanguineo;
    }
}
